wgpu/src/backend/direct.rs

```diff
@@ -1702,7 +1702,7 @@ impl crate::Context for Context {
             },
             callback: wgc::resource::BufferMapCallback::from_rust(Box::new(|status| {
                 let res = match status {
-                    wgc::resource::BufferMapAsyncStatus::Success => Ok(()),
+                    Ok(()) => Ok(()),
                     _ => Err(crate::BufferAsyncError),
                 };
```

Comment:
This now should just be `status.map_err(|_| crate::BufferAsyncError)`

Reply:
Done.